templates/corosync.j2

```diff
@@ -0,0 +1,49 @@
+totem {
+        version: {{ ha_cluster_corosync_params.totem.version | default(2) }}
+        token: {{ ha_cluster_corosync_params.totem.token | default(30000) }}
+        consensus: {{ ha_cluster_corosync_params.totem.consensus | default(36000) }}
+        token_retransmits_before_loss_const: {{ ha_cluster_corosync_params.totem.token_retransmits_before_loss_const | default(6) }}
+        secauth: {{ ha_cluster_corosync_params.totem.secauth | default("on") }}
+        crypto_cipher: {{ ha_cluster_corosync_params.totem.crypto_cipher | default("aes256") }}
+        crypto_hash: {{ ha_cluster_corosync_params.totem.crypto_hash | default("sha1") }}
+        clear_node_high_bit: {{ ha_cluster_corosync_params.totem.clear_node_high_bit | default("yes") }}
+        rrp_mode: {{ ha_cluster_corosync_params.totem.rrp_mode | default("passive") }}
+
+        interface {
+                ringnumber: 0
+                bindnetaddr: {{ ansible_facts['eth0']['ipv4']['address'] }}
+                mcastport: 5405
+                ttl: 1
+        }
+        transport: udpu
+}
+logging {
+        fileline: {{ ha_cluster_corosync_params.logging.fileline | default("off") }}
+        to_logfile: {{ ha_cluster_corosync_params.logging.to_logfile | default("yes") }}
+        to_syslog: {{ ha_cluster_corosync_params.logging.to_syslog | default("yes") }}
+        logfile: /var/log/cluster/corosync.log
+        debug: {{ ha_cluster_corosync_params.logging.debug | default("off") }}
+        timestamp: {{ ha_cluster_corosync_params.logging.timestamp | default("on") }}
+        logger_subsys {
+                subsys: QUORUM
+                debug: off
+        }
+}
+nodelist {
+{% for host in ansible_play_batch %}
+       node {
+               ring0_addr: {{ hostvars[host]['ansible_facts']['eth0']['ipv4']['address'] }}
+               {% if hostvars[host]['ansible_facts']['eth1']['ipv4']['address'] is defined %}
+               ring1_addr: {{ hostvars[host]['ansible_facts']['eth1']['ipv4']['address'] }}
```

Comment:
No IPv6 support? What if the network interfaces have other names than eth0, eth1 (I'm not sure if that can happen on Suse).

Reply:
This is initial setup for Suse so it does not account for all potential possibilities. I think they can be added subsequently, along with updated readme to account for corosync variable.

Reply:
OK, that makes sense. Start with the basics, add complexities and features later. :+1: